bubuku/features/restart_if_dead.py

```diff
@@ -23,9 +24,13 @@ def run(self, current_actions) -> bool:
         if self.broker.is_running_and_registered():
             return False
         _LOG.info('Waiting for complete death')
-        self.broker.stop_kafka_process()
+        if not self.stopped:
+            if not self.broker.stop_kafka_process():
+                return True
```

Comment:
well `run` returns flag - if it is needed to execute this action again, and this can not be done using exceptions (or it will look shitty). for starting process probably you are right - it's possible to use exception. But again it will be binary logic - try to start again or not. Perfectly matches to boolean.